tests/integration/test_github.py

```diff
@@ -36,6 +36,18 @@ def test_authorize(self):
 
         assert isinstance(auth, github3.auths.Authorization)
 
```

Comment:
hey @sigmavirus24 do we want to add (2) test cases for `test_add_email_address` and `test_add_email_addresses` ? Specifically if `addresses is None` then `self.session.post.called is not True` ?